.ci/setup_env_ubuntu.sh

```diff
@@ -42,6 +42,10 @@ declare -A packages=( \
 	[redis]="redis-server" \
 )
 
+if [ "${NAME}" == "Ubuntu" ] && [ "$(echo "${VERSION_ID} < 20.04" | bc -q)" == "1" ]; then
```

Comment:
I think we can remove the  `[ "${NAME}" == "Ubuntu" ]` , we are supposed to run this script only on ubuntu distro, right?